cypress/TestCases/FireboltCertification/Manage/ClosedCaptionsManage.feature

```diff
@@ -2,113 +2,114 @@ Feature: ClosedCaptions_Manage
+            | Set fontEdge-drop-shadow-left        | fontEdge           | drop_shadow_left      |
+            | Set fontEdge-drop-shadow-right       | fontEdge           | drop_shadow_right     |
+            | Set fontEdgeColor-#FFFFFF            | fontEdgeColor      | #FFFFFF               |
+            | Set fontFamily-cursive               | fontFamily         | cursive               |
+            | Set fontFamily-monospaced_sanserif   | fontFamily         | monospaced_sanserif   |
+            | Set fontFamily-proportional_serif    | fontFamily         | proportional_serif    |
+            | Set fontFamily-monospaced_serif      | fontFamily         | monospaced_serif      |
+            | Set fontFamily-proportional_sanserif | fontFamily         | proportional_sanserif |
+            | Set fontFamily-smallcaps             | fontFamily         | smallcaps             |
+            | Set fontFamily-casual                | fontFamily         | casual                |
+            | Set fontOpacity-75                   | fontOpacity        | 75                    |
+            | Set fontSize-0.5                     | fontSize           | 0.5                   |
+            | Set fontSize-1                       | fontSize           | 1                     |
+            | Set fontSize-1.5                     | fontSize           | 1.5                   |
+            | Set textAlign-left                   | textAlign          | left                  |
+            | Set textAlign-center                 | textAlign          | center                |
+            | Set textAlign-right                  | textAlign          | right                 |
+            | Set textAlignVertical-top            | textAlignVertical  | top                   |
+            | Set textAlignVertical-bottom         | textAlignVertical  | bottom                |
+            | Set preferredLanguages               | preferredLanguages | spa,eng               |
+            | Set windowColor-white                | windowColor        | white                 |
+            | Set windowOpacity-50                 | windowOpacity      | 50                    |
 
     @ClosedCaptions @manageSDK
-    Scenario Outline: Closedcaptions.<Method> - Positive Scenario: <Scenario> with 'null' params
-        When 1st party app registers for the '<Event>' event using the 'Firebolt' API
-        And 1st party app invokes the 'Firebolt' API to '<Set_API_Key>'
-        Then 'Firebolt' platform responds to '1st party app' for '<Set_API_Key>'
-        When 1st party app invokes the 'Firebolt' API to '<API_Key>'
-        Then 'Firebolt' platform responds to '1st party app' with '<Method_Validation_Key>'
-        And 'Firebolt' platform triggers to '1st party app' with '<Event_Validation_Key>'
+    Scenario Outline: ClosedCaptions.<Method> - Positive Scenario: <Scenario>
+        Given we test the 'CLOSED_CAPTIONS_SETTINGS' getters and setters '<Method>' to '<Value>'
```

Comment:
We reference Value but that doesn't exist in the examples?

Reply:
update the step adding the value as null